packages/auth-client/src/client.ts

```diff
@@ -3,9 +3,9 @@ import {
   generateChildLogger,
   getDefaultLoggerOptions,
   getLoggerContext,
+  pino,
 } from "@walletconnect/logger";
 import { EventEmitter } from "events";
```

Comment:
I remember the reason we installed `pino` explicitly was because of some random-ish build errors, do these not happen anymore?

Reply:
These are fixed and shouldn't happen anymore